docs/source/conventions.rst

```diff
@@ -60,14 +62,13 @@ If a bug is relatively quick and easy to handle, we call it a hotfix. A hotfix i
 
 .. note::
 
-    If you work on a project using Nansat, this project should use the master version of Nansat. Two
+    If you work on a project using Nansat, this project should use the master version of Nansat. Different
     situations may then occur:
 
-    1. You discover a **bug**. This can be solved with a **hotfix**.
-    2. You want to add functionality to Nansat. This is solved by creating an issue in Nansat and a
-       related issue branch. When this is completed and merged into develop, we may have a new
-       release.
-    3. You want to add a mapper. This can be done by adding a package ``nansat_mappers`` to your
+    1. You discover a **bug** that is easy to fix, this should be solved with a **hotfix** branched from master.
```

Comment:
I suggest rather "... fix. This should be solved in a **hotfix branch** originating from master. Same for points 2 and 3.